mlir/lib/Dialect/Transform/Transforms/TransformInterpreterUtils.cpp

```diff
@@ -121,23 +121,94 @@ ModuleOp transform::detail::getPreloadedTransformModule(MLIRContext *context) {
       ->getLibraryModule();
 }
 
+namespace {
+
+transform::TransformOpInterface
+findTransformEntryPointNonRecursive(Operation *op, StringRef entryPoint) {
```

Comment:
llvm prefers `static` functions over putting them in anonymous  namespaces.
```suggestion
static transform::TransformOpInterface
findTransformEntryPointNonRecursive(Operation *op, StringRef entryPoint) {
```

See https://llvm.org/docs/CodingStandards.html#restrict-visibility. In MLIR, we are less strict when it comes to keeping anonymous namespaces small, though.

Reply:
Thanks for the review! I've addressed your comments in this commit: https://github.com/llvm/llvm-project/pull/151323/commits/c204fd14ae715e47dbde7849f1ec5df866a6bc03